doc/install/install-openshift.md

```diff
@@ -0,0 +1,216 @@
+
+To install, run the following:
+
+```bash
+kubectl create ns istio-system
+```
+
+```bash
+kubectl  apply -f - <<EOF
+kind: OperatorGroup
+apiVersion: operators.coreos.com/v1
+metadata:
+  name: sail
+  namespace: istio-system
+spec: 
+  upgradeStrategy: Default  
+---  
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sailoperator
+  namespace: istio-system
+spec:
+  channel: 3.0-dp1
+  installPlanApproval: Automatic
+  name: sailoperator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+```
+
+To check the status of the install, you can run:
+
+```bash
+kubectl get installplan -n istio-system -o=jsonpath='{.items[0].status.phase}'
+```
+
+Once ready it will be marked `complete` while installing it will be marked `installing`.
+
+#### Configure Istio
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.21.0
+  namespace: istio-system
+  # Disable autoscaling to reduce dev resources
+  values:
+    pilot:
+      autoscaleEnabled: false
+EOF
+```
+
+Wait for Istio to be ready:
+
+```bash
+kubectl wait istio/default -n istio-system --for="condition=Ready=true"
+```
+
+
+### (Recommended) Thanos and Observability Stack
+
+Kuadrant provides a set of sample dashboards that use the known metrics exported by Kuadrant and Gateway components to provide insight into the different areas of your APIs and Gateways. While this is not essential, it is recommended that you set up an observability stack. Below are links to the OpenShift docs on this and also a link to help with the setup of Thanos for metrics storage.
+
+OpenShift supports a user facing monitoring stack. This can be cofigured and setup this documentation:
+
+https://docs.openshift.com/container-platform/latest/observability/monitoring/configuring-the-monitoring-stack.html
+
+If you have user workload monitoring enabled. We Recommend configuring remote write to a central storage system such as Thanos:
+
+- [Remote Write Config](https://docs.openshift.com/container-platform/latest/observability/monitoring/configuring-the-monitoring-stack.html#configuring_remote_write_storage_configuring-the-monitoring-stack)
+- [Kube Thanos](https://github.com/thanos-io/kube-thanos)
+
+There are a set of [example dashboards and alerts](https://docs.kuadrant.io/kuadrant-operator/doc/observability/examples/) for observing Kuadrant functionality.
+These dashboards and alerts make use of low level cpu, metrics and network metrics available from the user monitoring stack in Openshift. They also make use of resource state metrics from Gateway API and Kuadrant resources.
+To scrape these additional metrics, you can install a kube-state-metrics instance, with a custom resource config:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/config/observability/openshift/kube-state-metrics.yaml
```

Comment:
note the `main` in the url will not work until this PR lands as the file referenced is new in this pr.

If trying this out now, you can use this:

```sh
kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/install-doc/config/observability/openshift/kube-state-metrics.yaml
```